app/templates/layouts/_base.html

```diff
@@ -114,6 +121,8 @@
 
     <!--[if gte IE 9]><!-->
     {% block scripts %}
+      {% set scripts_cdn = cdn_url_prefix~"/scripts/bundle.min.js" %}
+      <script src="{{ scripts_cdn }}"></script>
       <script src="{{ url_for('static', filename='js/bundle.js') }}"></script>
```

Comment:
Do we have 2 called `bundle`? one on the CDN and one with the app?

Reply:
We certainly do
...for now

Reply:
The same goes for styles